### ISSUE TITLE
Is there any ListBox widget here?

### ISSUE THREAD

**Issue by jedi2light** [nucleic/enaml#335]

**Comment by frmdstryr** [comment]
Not currently, you can make one with a  [RawWidget](https://enaml.readthedocs.io/en/latest/api_ref/widgets/raw_widget.html?highlight=RawWidget)

**Comment by jedi2light** [comment]
Оh, that's what why I love open source!

**Comment by jedi2light** [comment]
Anyway, thank you for your quick reply!

**Comment by jedi2light** [comment]
Thank for your reply, is there any doc for this?

**Comment by MatthieuDartiailh** [comment]
I don't think so. The docstring is usually helpful and you can have a look at https://github.com/Exopy/exopy/blob/master/exopy/utils/widgets/qt_list_str_widget.py for an example.
